02-KernelModules_CharacterDevice/Makefile

```
@@ -0,0 +1,12 @@
+KDIR := /home/eduard/Kernel_i386
```

```suggestion
KDIR := /home/eduard/Kernel_i386
```
$KDIR refers to the path of the kernel source directory

Please use variable  as path

No hardcoded paths!